Build/start.sh

```diff
@@ -35,9 +35,6 @@ echo "IMAGE_INSTALL_append = \" gstreamer1.0-drpai opencv\"" >> ${WORK}/meta-mis
 #patch -p2 < ../rzv2l-drpai-conf.patch
 #echo "drpai patch applied"
 
-## Update number of CPUs in local.conf
-(NUM_CPU=$(nproc) && echo "BB_NUMBER_THREADS = \"$((NUM_CPU*2))\"" >> ${LOCALCONF}) || :
```

Comment:
This is pending but I['m running a test with the line changed to:
`(NUM_CPU=$(nproc) && echo "BB_NUMBER_THREADS = \"$((NUM_CPU))\"" >> ${LOCALCONF}) || :`
I think we still need `BB_NUMBER_THREADS ` to have parallelism, otherwise it's too slow

Reply:
On the page you linked, I read:
> The OpenEmbedded build system automatically configures this variable to be equal to the number of cores on the build system. For example, a system with a dual core processor that also uses hyper-threading causes the BB_NUMBER_THREADS variable to default to "4".

Where `dc02` has 8 XEON cores that support hyper-threading too